CONTRIBUTING.md

```diff
@@ -21,6 +21,13 @@ In order to run all of the tests we provide, you will need valgrind and clang.
 More specifically, under debian, you will need the package libc++-dev to
 properly run all the tests.
 
+Compiling and running grpc C++ tests depend on protobuf 3.0.0, gtest and gflags.
+Although gflags is provided in third_party, you will need to manually install
+that dependency on your system to run these tests. Under a Debian or Ubuntu
+system, you can install the gtests and gflags packages using apt-get:
+
+`apt-get install libgflags-dev libgtest-dev`
```

Comment:
Why is this in back-quotes, rather than using a shell command prompt like most of our install scripts? Also, should we be using the apt-get versions when we are already taking these as submodule dependencies?